src/HE.FMS.Middleware/HE.FMS.Middleware.csproj

```diff
@@ -5,24 +5,11 @@
     <OutputType>Exe</OutputType>
```

Comment:
Do we still need this project? remove?

Reply:
Yes we need it since there's no information regarding the mambu integration and still there's no decisions regarding SendToCrm function